categories.md

```diff
@@ -1,14 +1,15 @@
 ---
 layout: page
-title: Tags
+title: Categories
 ---
-{% assign tags =  site.recipes | map: 'tags' | join: ','  | split: ',' | uniq %}
+{% assign tags =  site.recipes | map: 'tags' | uniq | sort_natural %}
```

Comment:
There doesnt seem to be a need for joining and splitting by `,`. uniq does that natively